### PR TITLE
Split the deploy tasks on travis

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Fixes https://github.com/web-platform-tests/wpt.fyi/issues/1037

### CODE REVIEW

**Comment by lukebjerring** [comment]
Yeah, nesting the `env` lists under a single `if` didn't work (see 9c7974e2028e57457b047eece6191e4ebd363ba3).

File: .travis.yml [hunk 1]
```diff
@@ -28,20 +31,45 @@ matrix:
...
+      - DEPLOY_PR_STAGING_TARGET=webapp
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=results-processor
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=revisions/service
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=api/spanner/service
```
Comment (Hexcles) [inline .travis.yml]: We don't need this anymore, right? Let's stop deploying it.

File: .travis.yml [hunk 2]
```diff
@@ -28,20 +31,45 @@ matrix:
...
+      - MAKE_TEST_FLAGS="STAGING=true"
+    - if: (type = push) AND (branch = master)
+      env:
+      - DEPLOY_STAGING_TARGET=results-processor
+    - if: (type = push) AND (branch = master)
+      env:
+      - DEPLOY_STAGING_TARGET=revisions/service
+    - if: (type = push) AND (branch = master)
+      env:
+      - DEPLOY_STAGING_TARGET=api/spanner/service
```
Comment (Hexcles) [inline .travis.yml]: Ditto.

File: .travis.yml (hunk 1 above)
Comment (lukebjerring) [reply .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (lukebjerring) [reply .travis.yml]: Done.

**Comment by Hexcles** [comment]
LGTM

**Pull request merged**